pandas/tests/test_strings.py

```diff
@@ -26,6 +29,157 @@ def assert_series_or_index_equal(left, right):
         assert_index_equal(left, right)
 
 
+# method names plus minimal set of arguments to call
+_all_string_methods = [
+    ('get', [0]),
```

Comment:
this is somewhat duplicating: ``test_str_accessor_api_for_categorical``

can you reconcile and put in a pandas/conftest.py

*some* of the fixtures can stay here if they are specific to what is being tested. but the top level general stuff should move.

Reply:
I cannot find the test you're talking about - where is it supposed to be?

Do I understand correctly that you'd like the `_all_string_methods` in `pandas/conftest.py`? I would think that fixture is very specific to `test_strings`. The `dtype` fixtures make more sense on a higher level.

Reply:
```
(pandas) bash-3.2$ grep -r test_str_accessor_api_for_categorical pandas/tests/
Binary file pandas/tests//series/__pycache__/test_api.cpython-36-PYTEST.pyc matches
pandas/tests//series/test_api.py:    def test_str_accessor_api_for_categorical(self):
```

Reply:
Thanks. How about moving this test to `test_strings`? It's testing the API, but is clearly only related to the `str` accessor.